CAFMaker/prod_reco_caf_job.fcl

```diff
@@ -57,3 +57,4 @@ physics:
 }
 physics.producers.ssdclusts.FillTTree: false
 physics.producers.ssdclusts.CheckDQ: false
+physics.producers.cafmaker.GetMCTruth: false
```

Comment:
Same comment as in the fcl above. (If we have a line in this job fcl, it may make more sense to leave the default as true and then people can chose if they want to comment out this line or not.)

Reply:
commented out line and added comment of when its needed